src/kawaz/core/tests/runner.py

```diff
@@ -66,3 +67,5 @@ def setup_test_environment(self):
         # に引っかかる可能性が高いためOAuthのポスト部分を無効化する
         #
         settings.ACTIVITIES_ENABLE_OAUTH_NOTIFICATION = False
+        # 警告を無視
+        warnings.simplefilter("ignore")
```

Comment:
1. Reasons of the requirements of the warning suppression should be commented.
2. It should be more specific. Ignoring any warnings may hide important warnings should be fixed.